Git Testing/Service/APIPath.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  APIPath.swift
+//  Git Testing
+//
+//  Created by BrainX Technologies on 19/01/2021.
+//
+
+import UIKit
+
+enum APIPath:String {
+    case getPosts = ""
```

Comment:
path is empty ?

Reply:
if we need to implement Api then we can write end points in this class